docs/TROUBLESHOOTING.md

```diff
@@ -102,3 +102,21 @@ An error occurred (NotAuthorizedException) when calling the AdminConfirmSignUp o
 curl: (3) URL using bad/illegal format or missing URL```
 
 This happened when duplicate API gateways (i.e., `gateway_api_$ENV`) were created due to a Terraform state sync problem.
+
+### Cognito Authorizer Error
```

Comment:
I don't think we need to document this here, since we found the root cause of the issue and fixed it in #395.

Reply:
Updated!